editor/src/templates/editor.tsx

```diff
@@ -1,3 +1,6 @@
+// import feature switches so they are loaded before anything else can read them
+import '../utils/feature-switches'
```

Comment:
Why do we need this in both files?

Reply:
I got nervous that for the tests for example editor.tsx is more of an entry point, so putting it here "just in case".